cockatrice/src/user_context_menu.cpp

```diff
@@ -89,6 +94,27 @@ void UserContextMenu::banUser_processUserInfoResponse(const Response &r)
     dlg->show();
 }
 
+void UserContextMenu::adjustMod_processUserResponse(const Response &resp, const CommandContainer &commandContainer)
```

Comment:
^ Makes sense. If it fails to respond is that a big deal? Seems like that's how it's built now anyway